File: Modul-4/Soal-2/PRAK402-2410817210016-MUHAMMADRIZKYAKBAR.py
limit = int(input("Limit: "))

odd = [str(i) for i in range(1, limit + 1, 2)]
print(" ", " ".join(odd))

even = [str(i) for i in range(limit if limit % 2 == 0 else limit - 1, 1, -2)]
print(" ", " ".join(even))

File: Modul-4/Soal-2/PRAK402-2410817210016-MUHAMMADRIZKYAKBAR.c
#include <stdio.h>

int main(){
    
    int limit, i;

    printf("Limit: ");
    scanf("%d", &limit);

    printf("");
    for (i = 1; i <= limit; i += 2){
        printf("%d", i);
        printf(" ");
    }
    printf("\n");


    printf("");
    for (i = limit; i >= 2; i -= 2){
        printf("%d", i);
        printf(" ");
    }
    printf("\n");

    return 0;
}